tally_ho/apps/tally/views/reports/votes_per_candidate.py

```diff
@@ -50,7 +50,7 @@ def filter_queryset(self, qs):
             .values('candidate__full_name')\
             .annotate(total_votes_per_candidate=Sum('votes'))
 
-        keyword = self.request.GET.get('search[value]', None)
+        keyword = self.request.POST.get('search[value]', None)
```

Comment:
We can safely remove the `None` argument since the `get` method always returns `None` if the key value is not found.